GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -116,38 +117,34 @@ protected Tree(TreeNode treeNode, IGitUICommandsSource uiCommands)
                 TreeViewNode = treeNode;
             }
 
-            public Task ReloadAsync(CancellationToken token)
+            public async Task ReloadAsync(CancellationToken token)
             {
+                await TreeViewNode.TreeView.SwitchToMainThreadAsync(token);
```

Comment:
❓ Why not `this.SwitchToMainThreadAsync`?

Reply:
This class does not inherit from Control.